Packs/CTIX/Integrations/CTIX/CTIX.yml

```
@@ -0,0 +1,1150 @@
+        file_array = argToList(file_string)
+
+        for file in file_array:  # Check for Valid File Inputs
+            if not REGEX_MAP['hash'].match(file):
+                return_error('{0} is not a valid file.'.format(file))
+        enhanced = args.get('enhanced')
+        response = client.get_file_details(file_array, enhanced)
+        file_list = response.get("data", {}).get("results", {})
+        file_data_list, file_standard_list = [], []
+        for file_data in file_list:
+            score = to_dbot_score(file_data.get("score",0))
+            dbot_score = Common.DBotScore(
+                indicator=file_data.get("name2"),
+                indicator_type=DBotScoreType.FILE,
+                integration_name='CTIX',
+                score=score
+            )
+            file_standard_context = Common.File(
+                name=file_data.get("name2"),
+                dbot_score=dbot_score
+            )
+            file_standard_list.append(file_standard_context)
+            file_data_list.append(file_data)
+
+        readable_output = tableToMarkdown('File List', file_data_list, removeNull=True)
+
+        return CommandResults(
+            readable_output=readable_output,
+            outputs_prefix='CTIX.File',
+            outputs_key_field='name2',
+            outputs=file_data_list,
+            indicators=file_standard_list
+        )
+
+
+    def main() -> None:
+
+        base_url = demisto.params().get('base_url')
+        access_id = demisto.params().get('access_id')
+        secret_key = demisto.params().get('secret_key')
+
+        demisto.debug(f'Command being called is {demisto.command()}')
+        try:
+
+            client = Client(
+                base_url=base_url,
+                access_id=access_id,
+                secret_key=secret_key)
+
+            if demisto.command() == 'test-module':
+                test_module(client)
+            elif demisto.command() == 'ip':
+                return_results(ip_details_command(client, demisto.args()))
+            elif demisto.command() == 'domain':
+                return_results(domain_details_command(client, demisto.args()))
+            elif demisto.command() == 'url':
+                return_results(url_details_command(client, demisto.args()))
+            elif demisto.command() == 'file':
+                return_results(file_details_command(client, demisto.args()))
+
+        except Exception as e:
+            demisto.error(traceback.format_exc())  # print the traceback
+            return_error(f'Failed to execute {demisto.command()} command.\nError:\n{str(e)}')
+
+
+    if __name__ in ('__main__', '__builtin__', 'builtins'):
+        main()
+  type: python
+  commands:
+  - name: ip
+    arguments:
+    - name: ip
+      required: true
+      default: true
+      description: List of IPs.
+      isArray: true
+    - name: enhanced
+      auto: PREDEFINED
+      predefined:
+      - "True"
+      - "False"
```

```suggestion
      - "true"
      - "false"
```
usually, we call booleans in this way.
please change this also for all the other commands.